java/my-widget-service/src/main/java/software/amazon/awscdk/examples/MyWidgetServiceStack.java

```diff
@@ -0,0 +1,77 @@
+package software.amazon.awscdk.examples;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import software.amazon.awscdk.core.Construct;
+import software.amazon.awscdk.core.Duration;
+import software.amazon.awscdk.core.Stack;
+import software.amazon.awscdk.services.apigateway.LambdaIntegration;
+import software.amazon.awscdk.services.apigateway.LambdaIntegrationOptions;
+import software.amazon.awscdk.services.apigateway.Resource;
+import software.amazon.awscdk.services.apigateway.RestApi;
+import software.amazon.awscdk.services.apigateway.RestApiProps;
+import software.amazon.awscdk.services.iam.IManagedPolicy;
+import software.amazon.awscdk.services.iam.ManagedPolicy;
+import software.amazon.awscdk.services.iam.Role;
+import software.amazon.awscdk.services.iam.RoleProps;
+import software.amazon.awscdk.services.iam.ServicePrincipal;
+import software.amazon.awscdk.services.lambda.Code;
+import software.amazon.awscdk.services.lambda.Function;
+import software.amazon.awscdk.services.lambda.FunctionProps;
+import software.amazon.awscdk.services.lambda.Runtime;
+import software.amazon.awscdk.services.s3.Bucket;
+
+public class MyWidgetServiceStack extends Stack {
+	public MyWidgetServiceStack(final Construct scope, final String id) {
+		super(scope, id, null);
+
+		Bucket bucket = new Bucket(this, "WidgetStore");
+
+         RestApi api = new RestApi(this, "widgets-api",RestApiProps.builder()
+                 .restApiName("Widget Service")
+                 .description("This service serves widgets.")
+                 .build());
+
+		List<IManagedPolicy> managedPolicyArray = new ArrayList<IManagedPolicy>();
+		managedPolicyArray.add((IManagedPolicy) ManagedPolicy.fromAwsManagedPolicyName("AmazonS3FullAccess"));
+
+		Role restApiRole = new Role(this, "RestAPIRole",
+				RoleProps.builder().assumedBy(new ServicePrincipal("apigateway.amazonaws.com"))
```

Comment:
either split up the builder methods per line (preferred) or have them continuous, but be consistent.

Reply:
got it will fix it.